.gitlab-ci.yml

```diff
@@ -212,6 +212,9 @@ build-linux-image:
       - ARCH: [amd64, arm64, ppc64le]
   retry: 2
   script:
+    - |
+      DOCKER_PATH="artifactory:v2/cloud/role/docker-reader-role"
```

Comment:
Would it be better to define an anchor once and use aliases where needed?

```yaml
.docker-reader-role: &docker-reader-role
    'creds-helper init && eval $(creds-helper docker --eval "artifactory:v2/cloud/role/docker-reader-role")'

script:
  - *docker-reader-role
  -  make docker-otelcol ...
```

Reply:
It would be better! I didn't think of that. I will add to the PR.

Reply:
Trying this out on a separate gitlab build